libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss

```diff
@@ -0,0 +1,53 @@
+.card{
+    padding: 20px 30px;
+    background: #f4f4f4;
```

Comment:
Use variables in _variables.scss for colors, fonts etc

Reply:
Resolved the issue. Waiting for another review.